### PR TITLE
Add ability to include metadata in rpc calls.

### PR DESCRIPTION
**Pull request by muneebaahmad**
Issue: #2 

@yeuem1vannam thanks for the suggestion. This change works well for me!

@splittingred thanks for writing this gem. It is helping me a lot! Would you be able to review this PR and guide me on how to write a spec for this?

### CODE REVIEW

File: lib/gruf/rspec.rb [hunk 1]
```diff
@@ -45,12 +45,12 @@ module Rspec
   end
 
   config.before(:each, type: :gruf_controller) do
-    define_singleton_method :run_rpc do |method_name, request, &block|
+    define_singleton_method :run_rpc do |method_name, request, options = {}, &block|
```
Comment (splittingred) [inline lib/gruf/rspec.rb]: Can we do:

```ruby
define_singleton_method :run_rpc do |method_name, request, active_call_options: {}, &block|
```

That way we don't potentially collide if we need to pass other options later into the other called methods in that block.

Comment (muneebaahmad) [reply lib/gruf/rspec.rb]: Makes sense. Also, I think you meant `active_call_options = {}`. I just tested this change against my specs and it works well.

Comment (splittingred) [reply lib/gruf/rspec.rb]: I actually meant `active_call_options:` as a keyword argument; I'd like to keep optional arguments as keywords.

Comment (muneebaahmad) [reply lib/gruf/rspec.rb]: Got it. I made the changes and also tested them against my specs. Working well!

**Pull request merged**